lib/scripts/repo_export_in_batches.rb

```diff
@@ -36,6 +36,8 @@
   conversion_count = 0
 
   converter[:model_class].find_each do |record_for_export|
+    next if record_for_export.unexportable?
+
     converted_record = converter[:converter_class].new(
       record_for_export, collection_store.data, role_store.data
     )
```

Comment:
```suggestion
    converted_record = converter[:converter_class].new(
      record_for_export, collection_store.data, role_store.data
    )
    next if converted_record.skip?
```

Reply:
I would try moving the `#list_collections` collection_paths builder over to the GenericFile class. So we can still try to do the check before initializing the converter class. 

Then we can rename `#unexportable?` => `#skip?`